Library/Homebrew/test/.simplecov

```diff
@@ -20,7 +20,13 @@ SimpleCov.start do
     at_exit do
       exit_code = $!.nil? ? 0 : $!.status
       $stdout.reopen("/dev/null")
-      SimpleCov.result # Just save result, but don't write formatted output.
+
+      # Just save result, but don't write formatted output.
+      coverage_result = Coverage.result
+      SimpleCov.add_not_loaded_files(coverage_result)
+      simplecov_result = SimpleCov::Result.new(coverage_result)
```

Comment:
I don't think so. `SimpleCov.add_not_loaded_files(x)` just returns `x`, so I split it out of one long method chain for readability.